proto/node/akash/wasm/v1/genesis.proto

```
@@ -0,0 +1,17 @@
+syntax = "proto3";
+package akash.wasm.v1;
+
+import "gogoproto/gogo.proto";
```

Buf (1.61.0)

import "gogoproto/gogo.proto": file does not exist

(COMPILE)